numba/pretty_annotate.py

```diff
@@ -246,13 +246,12 @@ class Annotate:
     Function annotations persist across compilation for newly encountered
     type signatures and as a result annotations are shown for all signatures.
```

Comment:
I think this docstring might need an update now?

Reply:
Just updated to state that annotations for all signatures are shown by default, and included an example of using the `signature` keyword